docs/dev/plugins/cloudformation/cloudformation-basics.md

```diff
@@ -0,0 +1,42 @@
+# AWS CloudFormation
+How to transform a TOSCA model to an Amazon Web Services(AWS) CloudFormation template.
+
+## Mapping
+Mappings of TOSCA node types to AWS Resources are shown in [this table](mapping.md).
+
+## Transformation Lyfecycle
+
+### validate
+- Evironment check:
+    - AWS CLI installed
+- Return list of supported node types
+- Model check (properties):
+    - TBD
+
+### prepare
+- Add missing nodes to the graph to fulfill the requirements.
+- Split the graph, each part should representing a AWS Resource (multiple nodes can be put together to a single AWS Resource)
+
+### transform
+- Transform the graph/the combined nodes into a CloudFormation template
+
+### cleanup
+- TBD
```

Comment:
TBD? Maybe this should be resolved before merging this PR.

Reply:
No. I don't know what to do here yet. This will resolve while implementing

Reply:
Please replace it with: Will be addressed during implementation. Then this PR is good to go.